zerver/lib/exceptions.py

```diff
@@ -702,3 +702,16 @@ def __init__(self, setting_name: str) -> None:
     @override
     def msg_format() -> str:
         return _("'{setting_name}' must be a system user group.")
+
+
+class IncompatibleParameterValuesError(JsonableError):
+    data_fields = ["first_parameter", "second_parameter"]
+
+    def __init__(self, first_parameter: str, second_parameter: str) -> None:
+        self.first_parameter = first_parameter
+        self.second_parameter = second_parameter
```

Comment:
Not sure about the parameter names here. Maybe we can just pass a list like we do for `IncompatibleParametersError`, but there we use comma separated names. This way we can make sure only two parameters are passed.

Reply:
This is probably fine, easy to refactor later.